react/features/calendar-sync/components/ConferenceNotification.native.js

```diff
@@ -114,110 +105,34 @@ class ConferenceNotification extends Component<Props, State> {
      * @inheritdoc
      */
     render() {
-        const { event } = this.state;
         const { t } = this.props;
+        const { event } = this.state;
+        const customActionNameKey = [ 'notify.joinMeeting' ];
+        const customActionType = [ BUTTON_TYPES.PRIMARY ];
+        const customActionHandler = [ this._onGoToNext ];
+        const now = Date.now();
 
         if (event) {
-            const now = Date.now();
-            const label
-                = event.startDate < now && event.endDate > now
-                    ? 'calendarSync.ongoingMeeting'
-                    : 'calendarSync.nextMeeting';
+            const title = event.startDate < now && event.endDate > now
+                ? 'calendarSync.ongoingMeeting'
+                : 'calendarSync.nextMeeting';
+            const description = getLocalizedDateFormatter(event.startDate).fromNow();
 
             return (
-                <View
-                    style = { [
-                        styles.notificationContainer,
-                        this._getNotificationPosition()
-                    ] } >
-                    <View
-                        style = { this._getNotificationContentStyle() }>
-                        <TouchableOpacity
-                            onPress = { this._onGoToNext } >
-                            <View style = { styles.touchableView }>
-                                <View
-                                    style = {
-                                        styles.notificationTextContainer
-                                    }>
-                                    <Text style = { styles.notificationText }>
-                                        { t(label) }
-                                    </Text>
-                                    <Text style = { styles.notificationText }>
-                                        {
-                                            getLocalizedDateFormatter(
-                                                event.startDate
-                                            ).fromNow()
-                                        }
-                                    </Text>
-                                </View>
-                                <View
-                                    style = {
-                                        styles.notificationIconContainer
-                                    }>
-                                    <Icon
-                                        src = { IconArrowRight }
-                                        style = { styles.notificationIcon } />
-                                </View>
-                            </View>
-                        </TouchableOpacity>
-                    </View>
-                </View>
+                <Notification
```

Comment:
By doing this the notification is rendered outside of the notifications container, which is arguably wrong. Can you please refactor this to basically check if there is an ongoing meeting when a meeting is joined, and dispatch `showNotification` if necessary? That would probably require some work on a middleware or state listener.